nebula3/gclient/net/SessionPool.py

```diff
@@ -70,7 +70,8 @@ def __init__(self, username, password, space_name, addresses):
         self._close = False
 
     def __del__(self):
-        self.close()
+        if hasattr(self, '_lock'):
+            self.close()
```

Comment:
the root cause is  when init the pool if the address is invalid, would raise exception
so the object has no `_lock` property.

prefer move the exception at the end of __init__ function 
and the every object has `_lock` property.

Reply:
the current change it's also ok for me.
there's the same issue with meta client, please fix it in the same way